R/taxonomy.R

```diff
@@ -437,16 +444,24 @@ setMethod("getTaxonomyLabels", signature = c(x = "SummarizedExperiment"),
 #' @name hierarchy-tree
 #' 
 #' @examples
-#' # Generating a hierarchy tree based on available taxonomic information.
+#' # Generate a tree based on taxonomic rank hierarchy (a hierarchy tree).
 #' data(GlobalPatterns)
 #' tse <- GlobalPatterns
 #' getHierarchyTree(tse)
 #' 
-#' # Adding a hierarchy tree based on the available taxonomic information. 
+#' # Add a hierarchy tree to a TreeSummarizedExperiment.
 #' # Please note that any tree already stored in rowTree() will be overwritten.
 #' tse <- addHierarchyTree(tse)
 #' tse
 #' 
+#' # For instance, the TreeSummarizedExperiment Tengeler2020 contains an
+#' # actual phylogenetic tree.
+#' 
+#' data(Tengeler2020)
+#' tse2 <- Tengeler2020
+#' phylo_tree <- rowTree(tse2)
+#' phylo_tree
+#' 
```

Comment:
The message of this last point is somewhat unclear to me

The phylogeny tree looks very similar to calculated hierarchy tree. Moreover, phylo tree is dataset-specific. --> This last point can be removed if there is no clear message to tell